package.json

```diff
@@ -93,6 +92,7 @@
     "sinon": "^1.17.5",
     "sinon-chai": "2.8.0",
     "snazzy": "^5.0.0",
+    "style-attr": "^1.0.2",
```

Comment:
I believe this is a hard dependency.

Reply:
Ahh, I could have missed something, however the only reference of `style-attr` that I found was commented, found here (`src/utils/styleParser.js`):

https://github.com/aframevr/aframe/blob/master/src/utils/styleParser.js#L3

Reply:
I see, it might be able to just be removed then.

Reply:
removed it